torchgeo/datasets/fair1m.py

```diff
@@ -156,15 +156,81 @@ class FAIR1M(NonGeoDataset):
         "Bridge": {"id": 36, "category": "Road"},
     }
 
+    filename_glob = {
+        "train": os.path.join("train", "**", "images", "*.tif"),
+        "val": os.path.join("validation", "images", "*.tif"),
+        "test": os.path.join("test", "images", "*.tif"),
+    }
+    directories = {
+        "train": (
+            os.path.join("train", "part1", "images"),
+            os.path.join("train", "part1", "labelXml"),
+            os.path.join("train", "part2", "images"),
+            os.path.join("train", "part2", "labelXml"),
+        ),
+        "val": (
+            os.path.join("validation", "images"),
+            os.path.join("validation", "labelXml"),
+        ),
+        "test": (os.path.join("test", "images")),
```

Comment:
Working on a PR now to add type hints to a bunch of stuff. Just noticed this bug. Here `test` is of type `str`, not `tuple[str]`. Will fix in my other PR.